QuestionBot/QuestionBot.UnitTests/Model/StoreTests.cs

```diff
@@ -1,13 +1,77 @@
-﻿using NUnit.Framework;
+﻿using System;
+using System.Linq;
+using NUnit.Framework;
+using System.Collections.Generic;
+using QuestionBot.Model;
 
 namespace QuestionBot.UnitTests.Model {
 
     [TestFixture]
-    public class StoreTests {
+    public class StoreTests
+    {
+        private IStore _storeTest;
+        private IEnumerable<IRecord> _retreivedRecords = new List<IRecord>();
 
+    [SetUp]
+    public void Setup(){
+        _storeTest = new InMemoryStore();
+    }
+        [Test]
+        public void New_question_creates_record()
+        {
+            string question = "What is 1+2?";
+            IRecord questionRecord = _storeTest.CreateRecord(question);
+            Assert.AreEqual(questionRecord.Question, question);
+            Assert.AreNotEqual(questionRecord.TimeAsked, DateTime.MinValue);
+            Assert.AreNotEqual(questionRecord.ID,0);
```

Comment:
Good explanation!

I like that you're trying to validate that the TimeAsked and ID are being properly set and you're not using magic numbers to do this (for example, asserting ID = 1 is a flaky test). Since in your implementation ID can't be 0 and TimeAsked likely won't be `DateTime.MinValue` (unless you purposefully change your system time) this is probably not going to fail ever.

However, these are not ideal tests because asserting that `ID != 0` and `TimeAsked != DateTime.MinValue` doesn't mean that it's working properly, it means that it's not broken in this specific way. Ideally we could make a test that asserts that we get a unique ID and that TimeAsked is correct. We can't do this with the current implementation but it's OK to change the implementation to make something more testable. In this case though, since the implementation is so simple, it's not worth doing this. But something to keep in mind for future tests :)

Also note that you can get the default value of any type by doing `default(type)`. So you can change your lines to:

```cs
Assert.AreNotEqual(questionRecord.TimeAsked, default(DateTime));
Assert.AreNotEqual(questionRecord.ID, default(int));
```